addons/sourcemod/scripting/freak_fortress_2.sp

```diff
@@ -4103,10 +4103,10 @@ stock RemovePlayerBack(client, indices[], length)
 stock FindPlayerBack(client, index)
 {
 	new entity=MaxClients+1;
-	while((entity=FindEntityByClassname2(entity, "tf_wearable"))!=-1)
+	while((entity=FindEntityByClassname2(entity, "tf_wearable*"))!=-1)
```

Comment:
Can you walk me through this?  Is the entity actually named `tf_wearable*` now, or is the `*` the wildcard character?

Reply:
wildcard character. The Razorback is now called tf_wearable_razorback, before it was tf_wearable